main.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/JaSei/pathutil-go"
+	"github.com/alecthomas/kingpin"
+	"github.com/valyala/fasthttp"
+)
+
+var (
+	dir = kingpin.Flag("dir", "prefix path").Required().ExistingDir()
+)
+
+func main() {
+	kingpin.Parse()
+
+	log.Fatal(fasthttp.ListenAndServe(":8080", func(ctx *fasthttp.RequestCtx) {
```

Comment:
I would prefer to have the port also configurable via cmd parameter.

Reply:
good point :+1: